test/spec/LoginRouter_spec.js

```diff
@@ -544,7 +540,8 @@ function (Okta, Q, Backbone, xdomain, SharedUtil, CryptoUtil, CookieUtil, OktaAu
             origin: 'https://foo.com',
             data: {
               id_token: VALID_ID_TOKEN,
-              state: OIDC_STATE
+              state: OIDC_STATE,
+              nonce: OIDC_NONCE
```

Comment:
I missed this when it was introduced. Can we talk about it tomorrow so that I can catch up :)